amq/amq62-basic.json

```diff
@@ -18,62 +18,72 @@
         {
             "description": "The name for the application.",
             "name": "APPLICATION_NAME",
+            "displayName": "Application Name",
```

Comment:
Can you move this up to just above "description"? IMHO this would make it a bit cleaner.